qwc-front.php

```diff
@@ -60,6 +60,27 @@ function qwc_add_filters_front() {
 	//foreach ( $url_filters as $name => $priority ) {
 	//	add_filter( $name, 'qtranxf_convertURL', $priority );
 	//}
+
+	/* Fix the product categories and tags (displayed above the "additional informations" tab) */
+	add_filter( 'wp_get_object_terms', function ( $terms ) {
+		foreach ( $terms as $term ) {
+			if ( $term->taxonomy == 'product_cat' || $term->taxonomy == 'product_tag' ) {
+				$term->name = qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage( $term->name );
+			}
+		}
+
+		return $terms;
+	} );
+
+	/* Fix the product attributes (displayed in the "additional informations" tab) */
+	add_filter( 'woocommerce_attribute', function ( $text ) {
+		$values = explode( ', ', $text );
+		foreach ( $values as $i => $val ) {
+			$values[ $i ] = qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage( $val );
+		}
+
+		return implode( ', ', $values );
+	} );
 }
 qwc_add_filters_front();
```

Comment:
Ok, I tried. Global attributes work after re-saving, but "custom product attributes" do not work yet, even after adding "[:]":
![image](https://cloud.githubusercontent.com/assets/362092/6422672/90e1f44a-bedb-11e4-8858-96c27b4d0efa.png)

Here's what I entered for the custom attributes: 
![image](https://cloud.githubusercontent.com/assets/362092/6422685/b29012c0-bedb-11e4-8532-765c2a13873f.png)

I'll investigate more tomorrow. But even if it works after adding the "[:]" I need to think about a migration path, because the custom attributes are stored in a serialized array, so they can't easily be updated.